text/0039-reduce-module-size.md

```diff
@@ -0,0 +1,534 @@
+# Reduce aws-cdk-lib package size
+
+* **Original Author(s):**: @madeline-k
+* **Tracking Issue**: #39
+* **API Bar Raiser**: tbd
+
+The AWS CDK v2 framework library, `aws-cdk-lib` contains all packages from AWS
+CDK v1 in one monolithic package. While this is a great value for customers in
+making dependency management much easier, it has also resulted in increasing the
+unpacked npm package size to 234 MB, at the time of writing. This feature
+reduces the size of the package to under 100 MB.
```

Comment:
I was thinking this is a somewhat conservative estimate based on the size reduction we can expect from each section below. However, it is really hard to predict what the size will actually be after we've implemented it. We could either add an Appendix to explicitly justify why it will end up under 100 MB, or remove it entirely. What do you think?

Reply:
Removed, let me know if you disagree.